atomic_reactor/plugins/pre_pin_operator_digest.py

```diff
@@ -68,6 +72,18 @@ def __init__(self, tasker, workflow, replacement_pullspecs=None,
         self.replacement_pullspecs = replacement_pullspecs or {}
         self.operator_csv_modifications_url = operator_csv_modifications_url
 
+    def _validate_operator_csv_modifications_schema(self, modifications):
+        """Validate if provided operator CSV modification are valid according schema"""
+        schema = load_schema(
+            'atomic_reactor',
+            'schemas/operator_csv_modifications.json'
+        )
+        validate_with_schema(modifications, schema)
+
+    def _validate_operator_csv_modifications_schema(self, modifications):
```

Comment:
oops, name collision

Reply:
yup, I split patches wrongly, will be fixed